packages/matter-node.js/src/crypto/Crypto.ts

```diff
@@ -0,0 +1,195 @@
+/**
```

Comment:
Crypto libs should move to matter.js

Reply:
Yes - and will also get abstracted. On the plan